pkgs/top-level/all-packages.nix

```diff
@@ -6253,6 +6253,7 @@ with pkgs;
   scala_2_10 = callPackage ../development/compilers/scala/2.10.nix { };
   scala_2_11 = callPackage ../development/compilers/scala/2.11.nix { };
   scala_2_12 = callPackage ../development/compilers/scala { jre = jre8; };
+  scala_dotty = callPackage ../development/compilers/scala/dotty.nix { jre = jre8;};
```

Comment:
It's easier to find packages if derivation name and attribute name match. Maybe just `dotty`?

Reply:
The idea was to put it next to "standard" scala... But simply `dotty` is fine as well. 

I guess that it will disappear anyway with scala 3.0 coming out one day. Isn't it a problem to create such an ephemeral package? On the other hand, anyone using it should be aware of it...